### PR TITLE
docs(week-1): Week 1 updates documented

### PR DESCRIPTION
**Pull request by soham4abc**
## Project

React UI

### Changes

Week  1 update for Official coding period

### Notes

@GMishx @shaheemazmalmmd Please do have a look

### CODE REVIEW

File: docs/2022/ui/updates/soham/2022-06-24.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+The response from the `/jobs` endpoint was used in order to get the data from the backend server
+
+Response format of the API: <br/>
+![Screenshot from 2022-06-24 10-04-12](https://user-images.githubusercontent.com/63705023/175462930-c5da2b7f-1490-4e1d-b4b8-cb077e96b452.png)
+
+At first `MDBReact` datatable was used to render the data but further discussions on optimising the page the idea was scrapped and `X-Total-pages` header was used to get the data from the server in already paginated form.
+
+Final UI of the All Recent jobs page after the discussions looks like: <br/>
+
+![Screenshot from 2022-06-24 10-10-47](https://user-images.githubusercontent.com/63705023/175463583-d1901fe9-29e3-4d59-ac15-66703ec31572.png)
```
Comment (GMishx) [inline docs/2022/ui/updates/soham/2022-06-24.md:29]: Same for this image.

**Review by GMishx — CHANGES_REQUESTED** [review]

File: docs/2022/ui/updates/soham/2022-06-24.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## All Recent Jobs Page 
+
+_(June 16th, 2022)_
+
+Implementation of the All Recent Jobs page in the UI started.
+The response from the `/jobs` endpoint was used in order to get the data from the backend server
+
+Response format of the API: <br/>
+![Screenshot from 2022-06-24 10-04-12](https://user-images.githubusercontent.com/63705023/175462930-c5da2b7f-1490-4e1d-b4b8-cb077e96b452.png)
```
[the remark this answers is not in the document]
Comment (soham4abc) [reply docs/2022/ui/updates/soham/2022-06-24.md]: Resolved

File: docs/2022/ui/updates/soham/2022-06-24.md (hunk 1 above)
Comment (soham4abc) [reply docs/2022/ui/updates/soham/2022-06-24.md]: Resolved

File: docs/2022/ui/updates/soham/2022-06-24.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## All Recent Jobs Page
+
+_(June 16th, 2022)_
+
+Implementation of the All Recent Jobs page in the UI started.
+The response from the `/jobs` endpoint was used in order to get the data from the backend server
+
+Response format of the API: <br/>
+![Screenshot from 2022-06-24 10-04-12](../../../../../static/img/docs_img/docs_api_res.png)
```
Comment (GMishx) [inline docs/2022/ui/updates/soham/2022-06-24.md:23]:
```suggestion
![Screenshot from 2022-06-24 10-04-12](/img/docs_img/docs_api_res.png)
```

And please move images to a folder under `static/img/reactUI`for consistency

File: docs/2022/ui/updates/soham/2022-06-24.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+At first `MDBReact` datatable was used to render the data but further discussions on optimising the page the idea was scrapped and `X-Total-pages` header was used to get the data from the server in already paginated form.
+
+Final UI of the All Recent jobs page after the discussions looks like: <br/>
+
+![Screenshot from 2022-06-24 10-10-47](../../../../../static/img/docs_img/alljobs_ui_sample.png)
```
Comment (GMishx) [inline docs/2022/ui/updates/soham/2022-06-24.md:29]:
```suggestion
![Screenshot from 2022-06-24 10-10-47](/img/docs_img/alljobs_ui_sample.png)
```

Please move this image as well.

**Review by GMishx — CHANGES_REQUESTED** [review]

File: docs/2022/ui/updates/soham/2022-06-24.md (hunk 3 above)
Comment (soham4abc) [reply docs/2022/ui/updates/soham/2022-06-24.md]: Resolved

File: docs/2022/ui/updates/soham/2022-06-24.md (hunk 4 above)
Comment (soham4abc) [reply docs/2022/ui/updates/soham/2022-06-24.md]: resolved

File: docs/2022/ui/updates/soham/2022-06-24.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+## All Recent Jobs Page
+
+_(June 16th, 2022)_
+
+Implementation of the All Recent Jobs page in the UI started.
+The response from the `/jobs` endpoint was used in order to get the data from the backend server
+
+Response format of the API: <br/>
+![Screenshot from 2022-06-24 10-04-12](../../../../../static/img/reactUI/docs_api_res.png)
```
Comment (GMishx) [inline docs/2022/ui/updates/soham/2022-06-24.md:23]:
```suggestion
![Screenshot from 2022-06-24 10-04-12](/img/reactUI/docs_api_res.png)
```

File: docs/2022/ui/updates/soham/2022-06-24.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+At first `MDBReact` datatable was used to render the data but further discussions on optimising the page the idea was scrapped and `X-Total-pages` header was used to get the data from the server in already paginated form.
+
+Final UI of the All Recent jobs page after the discussions looks like: <br/>
+
+![Screenshot from 2022-06-24 10-10-47](../../../../../static/img/reactUI/alljobs_ui_sample.png)
```
Comment (GMishx) [inline docs/2022/ui/updates/soham/2022-06-24.md:29]:
```suggestion
![Screenshot from 2022-06-24 10-10-47](/img/reactUI/alljobs_ui_sample.png)
```

**Review by GMishx — CHANGES_REQUESTED** [review]

**Comment by soham4abc** [comment]
@GMishx  Changes made as per request

**Pull request merged**